src/libs/Middleware/Reauthentication.js

```diff
@@ -82,6 +82,7 @@ function Reauthentication(response, request, isFromSequentialQueue) {
             }
 
             request.resolve(data);
+            return data;
```

Comment:
Return response data in this middleware so we can chain the response with the `SaveResponseInOnyx` middleware